docs/source/contents/4_draft_mapping.rst

```diff
@@ -1662,6 +1662,453 @@ opaque:hostItem
+                <physicalAddress>
+                    <text>City: Cleveland</text>
+                    <text>County: Bradley County</text>
+                    <text>State: Tennessee</text>
+                </physicalAddress>
+            </holding>
+        </holdingExternal>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators/> .
+    @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+
+    <https://example.org/objects/1>
+        relators:rps "Cleveland State Community College" ;
+        skos:note "Shelf locator: Photograph Collection 2, People" .
+
+physicalLocation with holdingExternal
+-------------------------------------
+
+Use Case
+^^^^^^^^
+Some instances in our collections contain nested subelements for holdingExternal and the further nested physicalAddress information.
+
+Justification
+^^^^^^^^^^^^^
+To keep our metadata as simple as possible from a technical standpoint we will drop all information for holdingExternal. This type of information has little additive value when physicalLocation is already referenced.
+
+Xpath
+^^^^^
+
+:code:`mods:location/mods:holdingExternal`
+
+Decision
+^^^^^^^^
+We will drop all information for `holdingExternal`.
+
+`Example record from volvoices:2199 <https://digital.lib.utk.edu/collections/islandora/object/volvoices:2199/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation>University of Memphis. Special Collections</physicalLocation>
+        <holdingSimple>
+            <copyInformation>
+                <shelfLocator>Manuscript Number 5</shelfLocator>
+            </copyInformation>
+        </holdingSimple>
+        <holdingExternal>
+            <holding xsi:schemaLocation="info:ofi/fmt:xml:xsd:iso20775 http://www.loc.gov/standards/iso20775/N130_ISOholdings_v6_1.xsd">
+                <physicalAddress>
+                    <text>City: Memphis</text>
+                    <text>County: Shelby County</text>
+                    <text>State: Tennessee</text>
+                </physicalAddress>
+            </holding>
+        </holdingExternal>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators/> .
+    @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+
+    <https://example.org/objects/1>
+        relators:rps "University of Memphis. Special Collections" ;
+        skos:note "Shelf locator: Manuscript Number 5" .
+
+physicalLocation with @displayLabel="Address"
+---------------------------------------------
+
+Use Case
+^^^^^^^^
+
+Some of items with the physicalLocation of Pi Beta Phi Fraternity also have a physicalLocation subelement with the displayLabel attribute of Address.
+
+Justification
+^^^^^^^^^^^^^
+
+Similar to the holdingExternal, we will opt drop this information to maintain simplicity of our data from a technical standpoint.
+
+Xpath
+^^^^^
+
+:code:`mods:location/mods:physicalLocation[@displayLabel="Address"]`
```

Comment:
Ugh.  We may need to think about how this affects my suggestions above.

Reply:
Others note do make sense to be more specific. All `mods:location/mods:physicalLocation[@displayLabel="Address"]` should be dropped in all cases, so this would be correct, no?